src/Connection/Cluster/ClusterInterface.php

```diff
@@ -20,4 +21,11 @@
  */
 interface ClusterInterface extends AggregateConnectionInterface
 {
+    /**
+     * Executes given command on each connection from connection pool.
+     *
+     * @param  CommandInterface $command
+     * @return array
+     */
+    public function executeCommandOnEachNode(CommandInterface $command): array;
```

Comment:
This PR contains substantial changes. Please document everything in the `CHANGELOG.md`.

Reply:
@tillkruss Done